### PR TITLE
IDE-4139 add suport for breaking changne version

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by jtydhr88** [comment]
ci:retest

**Comment by gamerson** [comment]
Hey @jtydhr88  

This needs to be redesigned just a bit

screen shot 2018-08-07 at 11 20 23 pm

Two things

1) user needs to be able to import a Liferay workspace (that is at version 7.0 and needs to be upgraded to 7.1)
2) user needs to select the version of liferay they are upgrading to (7.1 or 7.0)

**Comment by gamerson** [comment]
Please make those changes and resend, thanks!

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/BreakingChangeVersion.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+package com.liferay.ide.project.ui;
+
+import java.util.stream.Stream;
+
+/**
+ * @author Simon Jiang
+ * @author Andy Wu
+ */
+public class BreakingChangeVersion {
```
[the remark this answers is not in the document]
Comment (simonjhy) [reply tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/BreakingChangeVersion.java]: Greg, you mean we don't need to set _upgradleLiferayWorkspace variable in BreakingChangeVersion?

[the remark this answers is not in the document]
Comment (simonjhy) [reply tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/BreakingChangeVersion.java]: I will remove this class

**Comment by simonjhy** [comment]
breaking change version can  be show when user want to upgrade a liferay workspace

**Pull request closed without merging**